windows/devices/uwb/UwbCxAdapter.cxx

```diff
@@ -0,0 +1,239 @@
+{
+    UwbSetAppConfigurationParametersBuilder builder{ uwbSessionData.sessionId };
+
+    // TODO NUMBER_OF_CONTROLEES
+    // TODO DEVICE_MAC_ADDRESS
+    // TODO DST_MAC_ADDRESS
+    // TODO DEVICE_TYPE
+    const auto sessionUwbMap = uwbSessionData.uwbConfiguration.GetValueMap();
+
+    for (const auto [ptag, value] : sessionUwbMap) {
+        std::visit(overloaded{
+                       [&builder, ptag](bool arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](uint8_t arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint8_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](uint16_t arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint16_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](uint32_t arg) {
+                           builder.Push(std::move(UwbAppConfigurationParameter{ uint32_t(arg), service_ddi_map.at(ptag) }));
+                       },
+                       [&builder, ptag](::uwb::protocol::fira::Channel arg) {
+                           uint8_t data = 9;
+                           switch (arg) {
+                           case (::uwb::protocol::fira::Channel::C5): {
+                               data = 5;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C6): {
+                               data = 6;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C8): {
+                               data = 8;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C9): {
+                               data = 9;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C10): {
+                               data = 10;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C12): {
+                               data = 12;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C13): {
+                               data = 13;
+                               break;
+                           }
+                           case (::uwb::protocol::fira::Channel::C14): {
+                               data = 14;
+                               break;
+                           }
+                           }
```

Comment:
If the value always maps to the logical channel number, then let's update the `enum class Channel` to have `uint8_t` as its base/underlying type, and explicitly set the corresponding values to what the spec defines, eg.

```
enum class Channel : uint8_t {
    C5 = 5,
    C6 = 6,
    C8 = 8,
    C9 = 9,
    C10 = 10,
    C12 = 12,
    C13 = 13,
    C14 = 14,
};
```

Then all of this code can be condensed to:
```suggestion
                           uint8_t data = notstd::to_underlying(arg);
```